app/controllers/comments_controller.rb

```diff
@@ -3,12 +3,12 @@
 # Controller Comments
 class CommentsController < ApplicationController
   skip_before_action :verify_authenticity_token
-  def create
+  before_action :correct_user?, only: %i[update destroy]
+  def create_comment
     @micropost = Micropost.find_by(id: params[:micropost_id])
-    @comment = @micropost.comments.build(content_comment: params[:micropost][:content_comment],
-                                         parent_comment_id: params[:parent_comment_id])
+    @comment = @micropost.comments.build(content: params[:micropost][:content], parent_id: params[:parent_id])
```

Comment:
Refractor like this format ( change parent_id params to params[:micropost][:parent_id]
@comment = @micropost.comments.build(comment_paranms)

    def comment_params
      params.require(:micropost).permit(%i[content parent_id])
    end